packages/swapper/src/api.ts

```diff
@@ -45,7 +45,13 @@ type ChainSpecificQuoteFeeData<T extends ChainId> = ChainSpecific<
 >
 
 export type QuoteFeeData<T extends ChainId> = {
+  minerFee: string // fee paid to the miner from the fee asset
+  tradeFeeBuyAsset: string // fee taken out of the trade from the buyAsset
+  tradeFeeSellAsset: string | undefined // fee taken out of the trade from the sellAsset
```

Comment:
Is there a reason for explicit undefined over `?` optional?

Reply:
Force of habit from a codebase where everything was explicit!

Made optional in https://github.com/shapeshift/lib/pull/1038/commits/6254c9d660d8f16e4fcc145453b7639ca52140b1